### PR TITLE
Adding module to manage Ruby Gem dependencies via Bundler

### PR DESCRIPTION
**Pull request by thoiberg**
I raised a PR for this last week but decided to add some additional arguments

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks for submitting this module to Ansible Extras. Apologies that it’s taken a while to get your module reviewed.

To help facilitate reviews, we’ve broadened the number of people who can approve modules for inclusion into the Extras repository. The list of official reviewers can be found here:

https://github.com/ansible/ansible-modules-extras/blob/devel/REVIEWERS.md 

Our new policy is that if a new module is reviewed and approved by at least two official module reviewers, the module will be approved for inclusion.  We will be asking the community of reviewers to take a look at these modules on a regular basis.

To ensure that your module has the best chance of being approved, please double-check that you adhere to the Ansible module guidelines: http://docs.ansible.com/developing_modules.html#module-checklist

File: packaging/language/bundler.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+- bundler: state=present executable=~/.rvm/gems/2.1.5/bin/bundle
+
+# Excludes the production group from installing
+- bundler: state=present exclude_groups=production
+
+# Only install gems from the default and production groups
+- bundler: state=present deployment=yes
+
+# Installs gems using a Gemfile in another directory
+- bunlder: state=present gemfile=../rails_project/Gemfile
```
Comment (ghost) [inline packaging/language/bundler.py]: typo

**Comment by ghost** [comment]
@thoiberg I tested this out and it works well. I like all of the bundler options support. If you fix the typo on the bundler line I marked, I will give you a plus one to get you closer to a merge.

**Comment by thoiberg** [comment]
@rickmendes Oops, maybe it's time to switch to an IDE with spell check :). I've fixed up the typo, thanks for reviewing it for me.

**Comment by ghost** [comment]
This module works for me, so I recommend it be merged. :+1: 

@gregdek Know any other Ruby guys that can review this so we can get it across the finish line?

**Comment by gregdek** [comment]
@rickmendes No, but we'll put out another broad call for module reviewers this week. If we don't get some action from that, we'll reconsider our options.

**Pull request merged**